src/treetensornetworks/abstracttreetensornetwork.jl

```diff
@@ -420,3 +420,25 @@ function inner(
   end
   return O[]
 end
+
+function expect(
+  operator::String,
+  state::AbstractTTN;
+  vertices=vertices(state),
+  root_vertex=default_root_vertex(siteinds(state)), #ToDo: verify that this is a sane default
+)
+  # ToDo: for performance it may be beneficial to also implement expect! and change the orthogonality center in place
+  # assuming that the next algorithmic step can make use of the orthogonality center being moved to a different vertex
+  sites = siteinds(state)
+  ordered_vertices = reverse(post_order_dfs_vertices(sites, root_vertex)) #ToDo: Verify that this is indeed the correct order for performance
+  ElT = promote_itensor_eltype(state)
+  res = Dictionary(vertices, Vector{ElT}(undef, length(vertices)))
+  for v in ordered_vertices
+    !(v in vertices) && continue  #only compute expectation values for required vertices
+    state = orthogonalize(state, v)
+    @assert isone(length(sites[v]))
+    Op = ITensors.op(operator, only(sites[v])) #ToDo: Add compatibility with more than a single index per vertex
```

Comment:
```suggestion
    Op = op(operator, only(sites[v])) #ToDo: Add compatibility with more than a single index per vertex
```
Does that work? Also I would change `Op` to `op_v`, since we generally favor lower case variable names.

Reply:
Also I think it is better to put the comment above the line of code, not appended to the end.